hooks/tk-multi-data-validation/basic/scene_operations.py

```diff
@@ -58,11 +58,17 @@ def register_scene_events(self, reset_callback, change_callback):
             )
         )
 
-        # Define the list of Alias event to that will trigger the change callback.
+        # Define the list of Alias event to that will trigger the change
+        # callback.
+        # NOTE: turning off the following change events, since Alias does not
+        # optimize emitting events, and so listening for these events will
+        # severely degrade performance:
+        # - DagNodeDeleted
+        # - DagNameModified
         change_event_names = [
             "DagNodeModified",
-            "DagNodeDeleted",
-            "DagNameModified",
+            # "DagNodeDeleted",
+            # "DagNameModified",
```

Comment:
In the future, we can add a setting defaulted to False.

Reply:
Yes we can add that in the future if customers want to specify which events they listen to. For now, this is the recommended default set of events.